cmd/podman/save.go

```diff
@@ -52,7 +41,7 @@ func init() {
 	saveCommand.SetUsageTemplate(UsageTemplate())
 	flags := saveCommand.Flags()
 	flags.BoolVar(&saveCommand.Compress, "compress", false, "Compress tarball image layers when saving to a directory using the 'dir' transport. (default is same compression type as source)")
-	flags.StringVar(&saveCommand.Format, "format", "", "Save image to oci-archive, oci-dir (directory with oci manifest type), docker-dir (directory with v2s2 manifest type)")
+	flags.StringVar(&saveCommand.Format, "format", "docker-archive", "Save image to oci-archive, oci-dir (directory with oci manifest type), docker-dir (directory with v2s2 manifest type)")
```

Comment:
The default here isn't one of the listed options?

Reply:
Nope, thats the way the help was dude.  But, if you look at the help, cobra does do this which to me is the best of all worlds:

```--format string   Save image to oci-archive, oci-dir (directory with oci manifest type), docker-dir (directory with v2s2 manifest type) (default "docker-archive")
```